spec/controllers/v2/concerns/index_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
```

Comment:
Let's name this file `collection_spec`, the name of the example can stay as it is